retracedhq/retraced
# Export logs in real time using vector

# This is an Enterprise feature, where retraced export audit logs in real-time to DataDog using vector.

## How to test:
- configure DataDog API key to `EXPORT_VECTOR_DATADOG_API_KEY` in `.env`
- run the service using `docker-compose up -d`
- send audit logs
- logs should appear in your DataDog logs section
